<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BindingGenerator</class>
 <widget class="QMainWindow" name="BindingGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BindingGenerator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Binding Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbBindingType">
        <item>
         <property name="text">
          <string>BC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UC</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Number of Targets</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbNumTargets">
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Target size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbTargetSize">
        <item>
         <property name="text">
          <string>Small</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Large</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGenFile">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Generated Files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbGenImages">
        <property name="text">
         <string>Generate Images</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
